.github/workflows/build-testbenches.yml

```diff
@@ -0,0 +1,61 @@
+name: Build testbenches
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  CROSS_CC_PREFIX: riscv-none-elf-
+  CROSS_CC_URL: https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download/v14.2.0-1/xpack-riscv-none-elf-gcc-14.2.0-1-linux-x64.tar.gz
+
+jobs:
+  build-testbenches:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install essential packages
+        run: |
+          sudo apt-get update -q -y
+          sudo apt-get upgrade -q -y
+          sudo apt-get install -q -y gcc-multilib g++-multilib
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: Test changed files
+        id: file-changed
+        uses: tj-actions/changed-files@v44
+        with:
+          fetch_additional_submodule_history: 'true'
+          files: |
+            tests/ansibench/*
+            tests/rv8-bench/*
+            tests/*.c
+      - name: Fetch xPack RISC-V GNU toolchain
+        if: ${{ steps.file-changed.outputs.any_changed == 'true' || github.event_name == 'workflow_dispatch' }}
+        run: |
+          mkdir -p /opt/riscv-gnu-toolchain
+          wget -O - $CROSS_CC_URL | tar -zxf - --strip-components=1 -C /opt/riscv-gnu-toolchain
+          echo "/opt/riscv-gnu-toolchain/bin" >> $GITHUB_PATH
+          echo "CROSS_CC=/opt/riscv-gnu-toolchain/bin/$CROSS_CC_PREFIX""gcc" >> $GITHUB_ENV
+      - name: Build RISC-V executables
+        if: ${{ steps.file-changed.outputs.any_changed == 'true' || github.event_name == 'workflow_dispatch' }}
+        run: |
+          CROSS_CC=$CROSS_CC USE_PREBUILT=0 make build-testbenches
+      - name: Create tarballs
+        if: ${{ steps.file-changed.outputs.any_changed == 'true' || github.event_name == 'workflow_dispatch' }}
+        run: |
+          tar -zcf rv32emu-prebuilt.tar.gz -C build bin
```

Comment:
If we are agreed to put all of the executables inside `build`, then the `bin` is not needed anymore.